src/utils.ts

```diff
@@ -438,13 +438,14 @@ export function validateCurve(
  * Get a 4-byte-long `Uint8Array` from a numeric value.
  *
  * @param value - The value to convert to a `Uint8Array`.
+ * @param littleEndian - Whether to use little endian byte order.
  * @returns The `Uint8Array` corresponding to the `bigint` value.
  */
-export function numberToUint32(value: number) {
+export function numberToUint32(value: number, littleEndian = false) {
```

Comment:
I put these simple changes to existing logic in separate commits, just so that its more explicit